src/lib/Scenes/Search/Input.tsx

```diff
@@ -67,6 +68,7 @@ export const Input = React.forwardRef<TextInput, InputProps & TextInputProps>(
                   input.current.clear()
                   setValue("")
                   rest.onChangeText?.("")
+                  rest.onClear?.()
```

Comment:
Oh neat! The equivalent syntax on Swift would be `rest.onClear?()`. I'll have to adjust my internal code parser 😛